introduction.md

```diff
@@ -1 +1,13 @@
 Greetings! I am ksed. My favorite Code School path is full-stack web development in JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?